tools/package_connector.sh

```diff
@@ -3,12 +3,10 @@ set -euo pipefail
 
 project_dir="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
 readonly project_dir
```

Comment:
Is there a specific reason for the other variables to be 1-liners while `project_dir` is defined and made read-only in separate lines?

Reply:
I remember that the `shellcheck` tool complains if you do this in one line 🤷

Reply:
Yes, I think so, too, but the other vars seem to be OK. Maybe simpler expressions?